src/vectorchord/usage/indexing.md

```diff
@@ -22,6 +22,15 @@ $$);
 
 > [!NOTE]
 > - `options` are specified using a [TOML: Tom's Obvious Minimal Language](https://toml.io/) string. You can refer to [#Index Options](#indexing-options) for more information.
+> - When dealing with large table (> $10^7$ rows), it will cost huge time and memory for `[build.internal]`. You can refer to [External Index Precomputation](../advanced-features/external-index-precomputation) to have a better experience.
```

Comment:
```suggestion
> - When dealing with large table, it will cost huge time and memory for `build.internal`. You can refer to [External Index Precomputation](../advanced-features/external-index-precomputation) to have a better experience.
```